README.adoc

```diff
@@ -526,13 +526,42 @@ If your regular expression contains groups, you can also use the matched values
 relabel "request_uri" {
   from = "request"
   split = 2
-  
+
   match "^/(users|profiles)/[0-9]+" {
     replacement = "/$1/:id"
   }
 }
 ----
 
+### File Globs
+
+You can specify one or more wildcards in the source file names, in which case the wildcards will be resolved to the corresponding list of files.
+
+Given a config like this:
+
+[source,hcl]
+---
+namespace "test" {
+  source {
+    files = ["/var/log/nginx/*_access.log"]
```

Comment:
This actually poses an interesting challenge: Regular files that are specified in the configuration, don't need to exists when the exporter starts -- rather, the exporter will continually watch the (non-existent) file and start processing as soon as it appears.

Using glob expressions, this becomes much harder, and an unsuspecting user might assume that glob expressions configured here might cause a similar behaviour, and pick up new files that match the glob expression while the exporter is running (which it won't, since the expressions are evaluated only once on startup).

I get that dynamically watching for new files that match a glob pattern would massively complicate this new feature, and I'm not sure if adding it would actually be worth the effort (probably not, and could always be added later without breaking BC). However, it should still be mentioned in the README to avoid any confusion in unsuspecting users. ;)